pkg/config/config.go

```diff
@@ -1044,7 +1044,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.cgroup_dump_timeout", 30)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.cgroup_wait_list_size", 10)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.cgroup_output_directory", "")
-	config.BindEnvAndSetDefault("runtime_security_config.network.enabled", false)
+	config.BindEnvAndSetDefault("runtime_security_config.network.enabled", true)
```

Comment:
This enables the feature by default. Not being mentioned in the PR description, I suppose this isn't intentional?

Reply:
This is actually intentional, I will cc you into the slack conversation. I will ask for the PR to be edited to include a bit more details